publishing/filterAPI/json.go

```diff
@@ -627,9 +627,35 @@ func GetValidPUTFilterOutputWithDimensionsJSON() string {
 		  {
 			  "name": "age",
 			  "options": [
-			    "27", "28"
+			    "27", "27", "28"
```

Comment:
Is this a valid request body, won't the filter api respond with duplicate option?

Reply:
If this allows for duplicate options and collapses them so there is no duplication in our database then it might be worth passing in a list of options to this function and then it is more flexible to set up different tests (one with duplicate options and one without)?